tests/__init__.py

```diff
@@ -34,3 +36,14 @@ def clean():
     testcase.addCleanup(clean)
 
     return tempfile
+
+
+@nottest
+def generate_test_file(testcase, lines):
+    tempfile = get_tempfile(testcase)
+
+    with open(tempfile, 'w') as f:
```

Comment:
Don't leave `encoding` unspecified, as it's platform dependent and we don't want that.